packaging/python/Dockerfile

```diff
@@ -55,6 +55,14 @@ RUN curl -L -o openmpi-4.0.3.tar.gz  https://download.open-mpi.org/release/open-
     && ./configure --prefix=/nrnwheel/openmpi \
     && make -j install
 
+RUN curl -L -o MUSIC.zip https://github.com/INCF/MUSIC/archive/refs/heads/switch-to-MPI-C-interface.zip \
```

Comment:
Note: we can now rely on v1.2.0
Is it expected to ship `libmusic.{so|dylib}` in the wheel or is it  desired to have it built/enabled and it is the user's responsibility to provide it?

Reply:
Maybe both ?

Reply:
@mdjurfeldt can comment as well. My present opinion is to ship as little as possible of MUSIC in the wheel. If the user later provides it, then it should work.